core/src/main/java/io/questdb/cairo/O3PartitionJob.java

```diff
@@ -222,14 +224,15 @@ public static void processPartition(
                 assert srcTimestampFd != -1 && srcTimestampFd != 1;
 
                 int branch;
+                long mergeEquals = tableWriter.isDeduplicationEnabled() ? 1 : 0;
```

Comment:
add more comment description

Reply:
added comments in [cb5db05](https://github.com/questdb/questdb/pull/3524/commits/cb5db05a8642d8626d8abb8a0c8f2db9d2ff19a6)